package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -552,16 +564,59 @@ def _read_next_timestep(self):
 
         indices = np.zeros(self.n_atoms, dtype=int)
 
-        f.readline()  # ITEM ATOMS etc
-        for i in range(self.n_atoms):
-            idx, _, xs, ys, zs = f.readline().split()
+        atomline = f.readline()  # ITEM ATOMS etc
+        attrs = atomline.split()[2:]  # attributes on coordinate line
+        col_ids = {}  # column index of each attribute
```

Comment:
can just make this a one liner, `col_ids = {i: attr for i, attr in enumerate(attrs)}`